insoundz_api/src/enhancer.py

```diff
@@ -172,7 +172,7 @@ def _enhancement_start(self, api, src, dst, retention, preset, pbar):
 
     def enhance_file(
         self, src, no_download=False, dst=None, retention=None,
-        preset=None, status_interval_sec=DEFAULT_STATUS_INTERVAL_SEC,
+        preset=None, enable_stereo=False, status_interval_sec=DEFAULT_STATUS_INTERVAL_SEC,
```

Comment:
I think it's always better to add new arguments to the end. Otherwise it can break old implementaion.
For example if someone calls the old omplemetion of enhance_file with enhance_file(65, 'flat', '3).